### PR TITLE
Update 2.1 Architecture

### PR DESCRIPTION
**Pull request by Drutol**
> Unlike the above described solutions, the \textbf{embedded gateway} is not derived from the middleware concept.

How do we define middleware? One could say that all three kinds of devices act as in-between agents translating data, including the embedded gateway.

> As a result, any product being advertised as Industry 4.0 enabled must be OPC UA-capable somehow

Does OPC UA really have a monopoly on being Industry 4.0 compliant? You mentioned some solution from India lately... Assuming the range of Industry 4.0 solutions must all be interoperable I'd reword it to make it sound less strict as in:

"As a result, any product being advertised as Industry 4.0 enabled must be capable of working with range of standards such as OPC UA somehow"

### CODE REVIEW

**Comment by Drutol** [comment]
Related to #37

**Review by mpostol — APPROVED** [review]

**Comment by mpostol** [comment]
Contributes to #37
